### PR TITLE
Keylight integration

### PR DESCRIPTION
**Pull request by Spivaka**
Fixes https://github.com/demisto/server/issues/4880

Still need to validate it with a user with enough permissions

### CODE REVIEW

File: Integrations/integration-KeyLight.yml [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |
+    var server = params.server.replace(/[\/]+$/, '') + ':' + params.port + '/';
```
Comment (aviadl) [inline Integrations/integration-KeyLight.yml]: Are you sure that regex works ?
I suspect it has some problems, but i might be wrong

File: Integrations/integration-KeyLight.yml [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+            var values = args.values && JSON.parse(args.values);
+            if (command === 'kl-delete-record-attachments') {
+                values = [{type: 'DynamicRecordList', key: args.fieldId, value: '<Record><Id>' + args.attachmentId + '</Id></Record>'}];
+            }
+            args.values = createValuesFromArgs(values, args.id);
+        }
+        return replaceInTemplates(bodyDictionary[command], args);
+    }
+
+    var filterFields = ['field-path', 'filter-type', 'value'];
```
Comment (aviadl) [inline Integrations/integration-KeyLight.yml]: Dont you have this field above already

File: Integrations/integration-KeyLight.yml (hunk 1 above)
Comment (Spivaka) [inline Integrations/integration-KeyLight.yml]: It does.

**Comment by Spivaka** [comment]
Added the fromversion while fixing issues from SureScripts

**Comment by aviadl** [comment]
@Spivaka what is this PR waiting for ?

**Comment by liorkol** [comment]
@Spivaka Please fix ASAP. Need this for Surescripts

**Pull request merged**